src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -248,6 +247,144 @@ public static void FaultException_Throws_WithFaultDetail()
         Assert.True(false, "Expected FaultException<FaultDetail> exception, but no exception thrown.");
     }
 
+    [Theory]
```

Comment:
Yes we can use [Fact] here. I was trying to use [Theory] to combine the new tests but that doesn't work.